src/components/swap/SwapModalDetailRow.tsx

```diff
@@ -0,0 +1,38 @@
+import Row from 'components/Row'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { ReactNode } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { Label } from './SwapModalHeaderAmount'
+
+const Value = styled.span<{ color?: string }>`
```

Comment:
Please wrap ThemedText appropriately and pass `color` prop as string to it. Note that `Text` is a div, so it automatically resolves @tinaszheng comment.

Reply:
i made this change